cmd/common/oci.go

```diff
@@ -418,6 +419,7 @@ func runInstanceSpecsDetached(
 		runtimeParams.Set("id", spec.ID)
 		runtimeParams.Set("name", spec.Name)
 		runtimeParams.Set("tags", strings.Join(spec.Tags, ","))
+		runtimeParams.Set("node", strings.Join(spec.Nodes, ","))
```

Comment:
The "node" is referencing an already existing runtime parameter. So we sadly can't just rename that without breaking backward compatibility. But I was also thinking about adding at least an alias, since "nodes" makes way more sense.